src/modules/kvs/cache.c

```diff
@@ -441,25 +440,37 @@ const char *cache_entry_get_blobref (struct cache_entry *entry)
     return entry ? entry->blobref : NULL;
 }
 
+/* wrapper around cache_entry_destroy() for zhashx() */
+static void _cache_entry_destroy (void **arg)
```

Comment:
I think we've been avoiding prefixing functions with underbars.  Maybe just `cache_entry_destroy_wrapper()`?